os/arch/arm/src/armv7-m/mpu.h

```diff
@@ -160,6 +160,12 @@
 #define MPU_RASR_AP_RORO      (6 << MPU_RASR_AP_SHIFT)	/* P:RO   U:RO   */
 #define MPU_RASR_XN             (1 << 28)	/* Bit 28: Instruction access disable */
 
+#ifdef CONFIG_MPU_STACK_OVERFLOW_PROTECTION
+#define STACK_GUARD_ADDR_MASK	0xffffffe0	/* Bits N-31:  Used for MPU Region base addrese */
+#define MPU_REGION_ADDRESS_ALIGNMENT 32	/* MPU regions must be 32 byte address aligned */
```

Comment:
Can you pls rename these 2 defines to indicate it is related to stack gaurd? Maybe like MPU_STACK_REGION_ALIGNMENT ?

Reply:
The address is aligned because of MPU address alignment issue, so I think we can keep same or even move outside of stack overflow feature.